.github/workflows/push.yml

```diff
@@ -33,3 +33,5 @@ jobs:
         go test ./...
         go test -c github.com/google/nftables
         sudo ./nftables.test -test.v -run_system_tests
+        go test -c github.com/google/nftables/integration
+        sudo ./integration.test -test.v -run_system_tests
```

Comment:
The tests succeed when run with the `go` tool, but not on GitHub Actions.

I think the best fix is to change directory in this line:

```
(cd integration && sudo ../integration.test -test.v -run_system_tests)
```

Reply:
done